egs/wsj/s5/steps/libs/nnet3/xconfig/parser.py

```diff
@@ -33,6 +33,7 @@
         'lstmp-layer' : xlayers.XconfigLstmpLayer,
         'fast-lstm-layer' : xlayers.XconfigFastLstmLayer,
         'fast-lstmp-layer' : xlayers.XconfigFastLstmpLayer,
+        'stats-layer': xlayers.XconfigStatsLayer,
```

Comment:
stats-layer is almost always better and faster than lstm in previous experiments.